src/lib/booleans.js

```diff
@@ -0,0 +1,88 @@
+function negate(a) {
```

Comment:
In this file after every closing bracket you have a ;
which is not necessary, and you don't have it in numbers and strings,